tests/sourcepoint.spec.ts

```diff
@@ -15,3 +13,16 @@ generateCMPTests('Sourcepoint-frame', [
 ], {
     skipRegions: ["US", "GB"],
 });
+
+generateCMPTests('Sourcepoint-frame', [
+    "https://www.sueddeutsche.de/",
+    "https://www.derstandard.at/",
+    "https://www.heise.de/",
+    'https://www.n-tv.de/',
+], {
+    skipRegions: ["US", "GB", "FR"],
+    expectPopupOpen: false,
+    testOptIn: false,
+    testOptOut: false,
```

Comment:
If both testOptout and testOptin are off, will the test ever be run?

Reply:
We will still test for CMP detection and whether the popup is open.